store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -199,6 +203,22 @@ public CarbonWriterBuilder withLoadOptions(Map<String, String> options) {
     return this;
   }
 
+  /**
+   * To support the load options for sdk writer
+   *
+   * @param key   the key of load option
+   * @param value the value of load option
+   * @return updated CarbonWriterBuilder object
+   */
+  public CarbonWriterBuilder withLoadOption(String key, String value) {
+    Objects.requireNonNull(key, "key of table properties should not be null");
```

Comment:
this is load options, not table properties

Reply:
ok, done